src/components/Founder.tsx

```diff
@@ -0,0 +1,66 @@
+import { VerificationType } from 'helpers/requestSignature'
+import { useEffect } from 'preact/hooks'
+import Description from 'components/Description'
+import KetlLogo from 'icons/KetlLogo'
+import Messages from 'models/Messages'
+import classnames, {
+  alignItems,
+  display,
+  flexDirection,
+  gap,
+  justifyContent,
+  padding,
+  space,
+} from 'classnames/tailwind'
+import createFounderProof from 'helpers/createFounderProof'
+import isDataInMessage from 'helpers/isDataInMessage'
+import postWebViewMessage from 'helpers/postWebViewMessage'
+
+const container = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  justifyContent('justify-center'),
+  alignItems('items-center'),
+  space('space-y-2'),
+  padding('px-8'),
+  gap('gap-2')
+)
+
+export default function Founder() {
+  useEffect(() => {
+    const handleMessage = (message: unknown) => {
+      if (!isDataInMessage(message)) return
+      const { data } = message as {
```

Comment:
We shouldn't convert using `as` there if we have a helper to define a type, isn't it?

Reply:
We should make sure that `data` contains everything we need, without checking it we might end up having corrupted data here

Reply:
We can add validation and error output later